tests/conftest.py

```diff
@@ -115,7 +115,7 @@ def factory(request):
 def network(backend):
```

Comment:
I think we should get rid of the backend name here as it is not exposed in the client. Could it be converted to a boolean with an explicit name (for instance `client_debug` or `client_debug_enabled`) ?

This would allow to remove the TODO line 131.

Reply:
Maybe `client_debug_local` ?  As this relates to the 'no Substra platform deployment' mode.

Reply:
Seems good to me! 👍